vars/README.md

```diff
@@ -48,3 +48,6 @@ blueprints include:
   emulated or physical UE. Details documented
   [here](https://docs.aetherproject.org/master/onramp/blueprints.html#oai-5g-ran). 
 
+* `main-srsran.yml`: Configures the Core with an SRSRAN-based 5G RAN,
+  supporting another gNB option. Details documented
+  TODO:.
```

Comment:
Are you planning to open a PR in the aether-docs?

Reply:
yes, i need to update the blueprint for srsRAN.